bundles/org.openhab.binding.netatmo/README.md

```diff
@@ -503,10 +509,9 @@ All these channels are read only.
 ### Welcome and Presence Camera
 
 Warnings:
-- The URL of the live snapshot is a fixed URL so the value of the channel cameraLivePictureUrl / welcomeCameraLivePictureUrl will never be updated once first set by the binding.
-So to get a refreshed picture, you need to use the refresh parameter in your sitemap image element.
-- Some features like the video surveillance are accessed via the local network, so it may be helpful to set a static IP address
-for the camera within your local network.
+
+- The URL of the live snapshot is a fixed URL so the value of the channel cameraLivePictureUrl / welcomeCameraLivePictureUrl will never be updated once first set by the binding. So to get a refreshed picture, you need to use the refresh parameter in your sitemap image element.
```

Comment:
Please put each sentence on its own line.

Reply:
This was done to avoid a warning in the checks (blank line required after a list item, something like that).
Either 2 lines to help maintenance but a warning, or one line and no warning.
Let me know what you prefer.

Reply:
Just leave as is then. I've prefer to avoid the warning even if it might be misguided.